sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/configuration/StatsbeatConnectionString.java

```diff
@@ -27,7 +27,8 @@ public final class StatsbeatConnectionString {
 
     private static final Pattern pattern = Pattern.compile("^https?://(?:www\\.)?([^/.-]+)");
 
-    private static final Set<String> EU_REGION_GEO_SET = new HashSet<>(10);
+    private static final Set<String> EU_REGION_GEO_SET = new HashSet<>(12);
+    private static final Set<String> NON_EU_REGION_GEO_SET = new HashSet<>(33);
```

Comment:
You'll want to divide the initial capacity by the load factor (default is 0.75f) if you don't want this `Set` to resize during insertion. That said, since the key is a String and String's are comparable it may be more performance to use a `String[]` and `Arrays.binarySearch` (this would need a JMH benchmark to verify)

Reply:
good point, this hash set is not used much so I kept it as hash set, but fixed it to pass in all values to the HashSet